Platform/AlderlakeBoardPkg/Library/FspmUpdUpdateLib/FspmUpdUpdateLib.c

```diff
@@ -689,4 +689,13 @@ UpdateFspConfig (
       }
     }
   }
+#if FixedPcdGetBool(PcdFusaSupport) == 1
+  if (SiCfgData->FusaConfigEnable)
```

Comment:
To avoid potential KW issue, I think it need check if SiCfgData is NULL as below.
if ((SiCfgData != NULL) && (SiCfgData->FusaConfigEnable)) {

Reply:
Should be fixed now.